Bottom: f3a1b4278a5d4f47365049f1c5853bf36c4ea33c
Top:    d21f879983683d039c9b381971dd8cc441f1759b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 15:46:02 +0100

Refresh of attempt-at-supporting-arc-arc

---

diff --git a/src/polygon.c b/src/polygon.c
index 575721b..63cbe3f 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -750,7 +750,6 @@ ArcPoly (ArcType * a, Coord thick)
 
   if (2 * M_PI * a->Width * (1. - (double)delta / 360.) - thick < MIN_CLEARANCE_BEFORE_BISECT)
     {
-      printf ("Bisecting arc with delta angle %f\n", delta);
       int half_delta = a->Delta / 2;
 
       seg1 = seg2 = *a;
